web/src/PickNeighbourhoodMode.svelte

```diff
@@ -103,7 +109,35 @@
     return { modalFilters, deletedModalFilters, travelFlows };
   }
 
-  // TODO Hover on button and highlight on map
+  function fillColor(
+    selectedPrioritization: "none" | "area" | "simd",
+  ): DataDrivenPropertyValueSpecification<string> {
+    let color: DataDrivenPropertyValueSpecification<string> = "red";
+    if (selectedPrioritization == "none") {
+      color = "black";
+    } else if (selectedPrioritization == "simd") {
+      color = makeRamp(["get", "simd"], simdLimits, simdColorScale);
+    } else if (selectedPrioritization == "area") {
+      color = makeRamp(["get", "area_km2"], areaLimits, simdColorScale);
+    }
+    return [
+      "case",
+      ["==", ["feature-state", "highlight"], "yes"],
+      "yellow",
+      // @ts-expect-error MapLibre expression types are weird, but this really does work
```

Comment:
Another use of moving this into the <script> section is being able to do this type cast / TS override. We can't easily do it in the section below in Svelte 4.